Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -35,7 +43,10 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
             $closeIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_ARRAY_SQUARE_BRACE, $index);
 
             $token->override('(');
+            $tokens->registerFoundTokenKind('(');
             $tokens[$closeIndex]->override(')');
+            $tokens->registerFoundTokenKind(')');
```

Comment:
First I made the `overrideAt` solution, then think about leaving override alone and simply create new token and set it via `offsetSet`, but it couses performance drop and memory increased highly.
I will recreate the first solution and push it later. That way the `->registerFoundTokenKind` method will not be public any longer.
Nice idea ;)